applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -65,6 +66,14 @@
                                             <meta itemprop="datePublished" content="@video.trail.webPublicationDate.toString("yyyy-MM-dd")" />
                                             <meta itemprop="name" content="@video.metadata.webTitle" />
                                             <meta itemprop="uploadDate" content="@video.trail.webPublicationDate.toString("yyyy-MM-dd")" />
+
+                                            @video.mediaAtom.map { ma =>
+                                                @ma.duration.map { duration =>
+                                                    <meta itemprop="duration" content="@MediaAtom.isoDuration(duration)" />
+                                                }
+                                            }
+                                            @* TODO: Set meta thumbnail once we have better image data in model *@
```

Comment:
Right now the posterImage is only an `Option[String]` so it would not be reliable to run the standard frontend image processing on it. I have raised this with Tools team. Also the media atom model does not provide height and width so I don't have that for the metadata either. CC @clloyd